mm2src/mm2_db/src/indexed_db/indexed_db.rs

```diff
@@ -298,6 +296,51 @@ impl<Table: TableSignature> DbTable<'_, Table> {
         send_event_recv_response(&self.event_tx, event, result_rx).await
     }
 
+    /// Adds the given `item` if there are no items with the same `index`.
+    /// https://developer.mozilla.org/en-US/docs/Web/API/IDBObjectStore/add
+    ///
+    /// * `index` - the name of a corresponding `Table`'s field by which records will be searched.
+    /// * `index_value` - the value of the `index`, therefore the value of a corresponding `Table`'s field.
+    pub async fn add_item_or_ignore_by_unique_index<Value>(
```

Comment:
this, `count` and `count_all` functions can be defined without `pub`

Reply:
These functions are used at
https://github.com/KomodoPlatform/atomicDEX-API/blob/baf6a4fc448fa713a9751565895566ff376f628c/mm2src/coins/tx_history_storage/wasm/tx_history_storage_v2.rs#L204
https://github.com/KomodoPlatform/atomicDEX-API/blob/baf6a4fc448fa713a9751565895566ff376f628c/mm2src/coins/tx_history_storage/wasm/tx_history_storage_v2.rs#L73

Reply:
I see only `count` function is used. But that's really weird that when you remove `pub` from `count`, it can still compile :thinking:.